bindings/pyroot/pythonizations/test/rdataframe_makenumpy.py

```diff
@@ -147,7 +147,7 @@ def test_sliced_array(self):
         """
         Test correct reading of a sliced numpy array (#13690)
         """
-        table = np.array([[1,2], [3,4]])
+        table = np.array([[1,2], [3,4]], dtype="int64")
```

Comment:
Similar to the other test, should we maybe better use `df.Take['int64_t']` further down to be consistent with NumPy?